release_notes.md

```diff
@@ -7,6 +7,32 @@
 - Changed default pull interval to 1 day to enhance performance.
     <!-- - The last 7 days worth of feed is large and requires that the default request timeout values for `nginx` and `uwsgi` is tweaked for the first ingestion to complete. -->
 
+## Upgrade Instructions for v3.1.2
+
+Upgrade to the latest version may result in playbooks failing. Following steps help remove the connector and upgrade it to the latest version.
+
+1.	Connect to the FortiSOAR&trade; instance's Command Line Interface (CLI).
+
+2.	Delete the FortiGuard connector v3.1.1 directories and files. You may use the following command:
+
+    ```bash
+    rm -r /opt/cyops/configs/integrations/connectors/fortinet-fortiguard-threat-intelligence_3_1_1/
```

Comment:
any permission or sudo user needed to run these commands?

Reply:
Good point. Let me add that

Reply:
Yes, it should be 'root' user. I have requested abhishek to add it.